One test failing

```bash
$ make test
...
writing manifest file 'cassandra_range_repair.egg-info/SOURCES.txt'
running build_ext
test_ten_commands (tests.test_execution_counts.execution_count_tests) ... Traceback (most recent call last):
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 405, in <module>
    main()
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 401, in main
    repair(options)
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 279, in repair
    tokens = Token_Container(options)
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 36, in __init__
    self.get_host_tokens()
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 116, in get_host_tokens
    success, _, stdout, stderr = run_command(*cmd)
  File "/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py", line 194, in run_command
    cmd = " ".join(command)
TypeError: sequence item 4: expected string, int found
ERROR
test_Murmur3_format_length (tests.test_ranges.range_tests) ... ok
test_Murmur3_range_end_zero (tests.test_ranges.range_tests) ... ok
test_Murmur3_range_start_zero (tests.test_ranges.range_tests) ... ok
test_Murmur3_range_wrap (tests.test_ranges.range_tests) ... ok
test_Random_range_end_zero (tests.test_ranges.range_tests) ... ok
test_Random_range_start_zero (tests.test_ranges.range_tests) ... ok
test_Random_range_wrap (tests.test_ranges.range_tests) ... ok

======================================================================
ERROR: test_ten_commands (tests.test_execution_counts.execution_count_tests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jens/Development/src/cassandra_range_repair/tests/test_execution_counts.py", line 14, in test_ten_commands
    subprocess.check_output(cmd)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['/Users/jens/Development/src/cassandra_range_repair/tests/../src/range_repair.py', '--nodetool', '/Users/jens/Development/src/cassandra_range_repair/tests/mock_nodetool_script', '-s', '4', '-w', '2']' returned non-zero exit status 1

----------------------------------------------------------------------
Ran 8 tests in 0.084s

FAILED (errors=1)
make: *** [test] Error 1
```

I'm running Python 2.7.10. This might have been caught earlier if something like #36 was in place.

--- a/src/range_repair.py
+++ b/src/range_repair.py
@@ -238,7 +238,7 @@ def run_command(*command):
     :param command: the command to be run and all of the arguments
     :returns: success_boolean, command_string, stdout, stderr
     """
-    cmd = " ".join(command)
+    cmd = " ".join(map(str, command))
     logging.debug("run_command: " + cmd)
     proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     stdout, stderr = proc.communicate()
@@ -375,7 +375,7 @@ def main():
     parser.add_option("-H", "--host", dest="host", default=platform.node(),
                       metavar="HOST", help="Hostname to repair [default: %default]")
 
-    parser.add_option("-P", "--port", dest="port", default=7199,
+    parser.add_option("-P", "--port", dest="port", default=7199, type="int",
                       metavar="PORT", help="JMX port to use for nodetool commands [default: %default]")
 
     parser.add_option("-s", "--steps", dest="steps", type="int", default=100,

